packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -48,25 +51,23 @@ class FramesBarPlotly {
 
   static List<Data> createFPSTraces() {
     final Data traceCpuGood = Data(
-      y: [],
-      x: [],
-      type: 'bar',
-      stackgroup: 'one',
-      name: 'CPU',
-      hoverinfo: 'y+name',
-      marker: Marker(color: CPU_COLOR),
-      width: [0],
-    );
+        y: [],
+        x: [],
+        type: 'bar',
+        stackgroup: 'one',
+        name: 'CPU',
+        hoverinfo: 'y+name',
+        marker: Marker(color: CPU_COLOR),
+        width: [0]);
```

Comment:
nit: add back trailing commas here and in traceGpuGood.

Reply:
Done